docs/nima/pico/pico_intro.adoc

```diff
@@ -60,13 +71,76 @@ To enable Pico add the following dependencies to your project's `pom.xml` (see M
 
 == Usage
 
-Once you have planned how each of your resources should support Pico, you specify the Pico behavior in one of two ways:
+These are the basic steps to help you get started with using Helidon Pico. 
+
+NOTE: The Pico examples [link] provide detailed instructions for using Helidon Pico. 
+
+*Step 1: Update your pom.xml file*
+
+Put the following dependencies in your `pom.xml` or `gradle.build` file: 
+
+
+.Builder dependencies
+[source,java]
+----
+    
+    <groupId>io.helidon.pico</groupId>
+    <artifactId>helidon-pico-processor</artifactId>
+    <version>${helidon.version}</version>
+----
+
+.Compile-time dependency:
+[source,java]
+----
+
+  <dependency>
+    <groupId>io.helidon.pico</groupId>
+    <artifactId>helidon-pico-services</artifactId>
+    <version>${helidon.version}</version>
```

Comment:
this is a better way to refer to versions so you don't need to constantly have APLHA4 -> ALPHA6, M1, etc issues to deal with.